distribution/deb/src/main/packaging/init.d/elasticsearch

```diff
@@ -104,6 +101,12 @@ if [ -f "$DEFAULT" ]; then
 	. "$DEFAULT"
 fi
 
+# Elasticsearch configuration file (elasticsearch.yml)
```

Comment:
Like - I mean make it clear that we needed to resolve user properties before we could resolve this properly properly.